UI/CMakeLists.txt

```diff
@@ -123,13 +123,17 @@ elseif(APPLE)
 	endif()
 
 elseif(UNIX)
-        find_package(Qt5X11Extras REQUIRED)
+	find_package(Qt5Gui REQUIRED)
```

Comment:
Seems like this is indented with tabs, whereas the previous code was indented with spaces. Which one is preferred these days?

Reply:
The rest of the file is tabs, these lines seemed to be the only ones with spaces.

Reply:
I see, thanks for clarifying

Reply:
(Seems like I can't resolve this thread, feel free to resolve it!)